akka-stream/src/main/scala/akka/stream/Graph.scala

```diff
@@ -78,6 +78,12 @@ trait Graph[+S <: Shape, +M] {
    * less specific than attributes set directly on the individual graphs of the composite.
    */
   def addAttributes(attr: Attributes): Graph[S, M] = withAttributes(traversalBuilder.attributes and attr)
+
+  /**
+   * Get all attributes from this `Graph` which have added previously.
+   */
```

Comment:
I think the method name says all that needs to be said already and that this Scaladoc does not add anything, skip for the other overloads as well
```suggestion
```

Reply:
ok, I will remove it